app/src/layouts/settingsLayout/footer/footer.jsx

```diff
@@ -0,0 +1,144 @@
+  },
+  slack: {
+    id: 'SettingsLayout.footer.slack',
+    defaultMessage: 'Chat with us on Slack',
+  },
+  contactUs: {
+    id: 'SettingsLayout.footer.contactUs',
+    defaultMessage: 'Contact us',
+  },
+  epam: {
+    id: 'SettingsLayout.footer.epam',
+    defaultMessage: 'EPAM',
+  },
+  documentation: {
+    id: 'SettingsLayout.footer.documentation',
+    defaultMessage: 'Documentation',
+  },
+  privacyPolicy: {
+    id: 'SettingsLayout.footer.privacyPolicy',
+    defaultMessage: 'Privacy Policy',
+  },
+  rights: {
+    id: 'SettingsLayout.footer.rights',
+    defaultMessage: '© Report Portal {currentYear} All rights reserved',
+  },
+  build: {
+    id: 'SettingsLayout.footer.build',
+    defaultMessage: 'Build: {buildVersion}',
+  },
+});
+
+const FooterBase = ({ buildVersion, instanceType, date }) => {
+  const { formatMessage } = useIntl();
+
+  const currentYear = date.getFullYear();
+
+  const footerLeftItems = [
+    formatMessage(messages.build, {
+      buildVersion,
+    }),
+    formatMessage(messages.rights, {
+      currentYear,
+    }),
+  ];
+
+  const footerRightItems = [
+    {
+      link: referenceDictionary.rpGitHub,
+      message: formatMessage(messages.github),
+    },
+    {
+      link: referenceDictionary.rpSlack,
+      message: formatMessage(messages.slack),
+    },
+    {
+      link: referenceDictionary.rpEmail,
+      message: formatMessage(messages.contactUs),
+    },
+    {
+      link: referenceDictionary.rpEpam,
+      message: formatMessage(messages.epam),
+    },
+    {
+      link: referenceDictionary.rpDoc,
+      message: formatMessage(messages.documentation),
+    },
+    (instanceType === EPAM || instanceType === SAAS) && {
+      link: referenceDictionary.rpEpamPolicy,
+      message: formatMessage(messages.privacyPolicy),
+    },
+  ];
+
+  return (
+    <div className={cx('footer')}>
+      <div className={cx('footer-left-items')}>
+        {footerLeftItems.map((item, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <div key={`footer-left-item-${i}`} className={cx('footer-item')}>
+            {item}
+          </div>
+        ))}
+      </div>
+      <div className={cx('footer-right-items')}>
+        {footerRightItems.map((item, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <Fragment key={`footer-right-item-${i}`}>
+            {item && (
+              <div className={cx('footer-item')}>
+                <a href={item.link} target="_blank" className={cx('link')}>
+                  {item.message}
+                </a>
+              </div>
+            )}
+          </Fragment>
+        ))}
+      </div>
+    </div>
+  );
+};
+
+FooterBase.propTypes = {
+  buildVersion: PropTypes.string.isRequired,
+  instanceType: PropTypes.string.isRequired,
+  date: PropTypes.instanceOf(Date),
+};
+FooterBase.defaultProps = {
+  date: new Date(),
+};
+export const Footer = connect((state) => ({
```

Comment:
Clould you explain, why we are using `connect` here?
My suggestion is to rewrite it to hooks.
Connect works differently from useSelector, but in this case it does not metter.